tests/test_filters.py

```diff
@@ -8,9 +8,9 @@
 from belogging.filters import LoggerFilter, LoggerDuplicationFilter
 
 
-def create_record(name='test_logger', level='DEBUG', msg='foobar'):
+def create_record(name='test_logger', level='DEBUG', msg='foobar', args=None):
```

Comment:
replace `args=None` with `*args`

Reply:
it will require *args as the first parameter, we cant pass named parameters after *args

Reply:
indeed, then, that's ok the way it is